.github/workflows/ci.yaml

```diff
@@ -215,7 +215,7 @@ jobs:
           # Make executable, since artifact download doesn't preserve this
           chmod +x ./ruff ${{ steps.ruff-target.outputs.download-path }}/ruff
 
-          ruff-ecosystem check ./ruff ${{ steps.ruff-target.outputs.download-path }}/ruff --cache ./checkouts --output-format markdown | tee ecosystem-result-check
+          ruff-ecosystem check ./ruff ${{ steps.ruff-target.outputs.download-path }}/ruff --cache ./checkouts --output-format markdown --force-preview | tee ecosystem-result-check
```

Comment:
Alternatively, we could get fancy and set this flag if a pull request has the `preview` label. I'm not sure that's worth it.